src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,223 @@
+$blue: #04529D;
+$outer-circle-left-pos: 91%;
+$outer-circle-top-pos: -2%;
+
+$hex-circle-left-pos: -8%;
+$hex-circle-top-pos: -4%;
+
+.menu {
+    margin: 40px;
+    position: relative;
+}
+
+.outer-circle {
+    width: 80px;
+    height: 80px;
+    border-radius: 50px;
+    color: #fff;
+    position: absolute;
+    left: $outer-circle-left-pos;
+    border: 1px solid grey;  
+}
+
+.intro-outer-circle {
+    @extend .outer-circle;
+    background: #fff;  
+    top: $outer-circle-top-pos;     
+}
+
+.inner-circle {
+    width: 52px;
+    height: 52px;
+    border-radius: 50px;
+    font-size: 20px;
+    font-weight: 900;
+    color: #fff;
+    line-height: 50px;
+    text-align: center;
+    position: absolute;
+    left: 17%;
+    top: 18%;
+    border: 1px solid grey;  
+}
+
+.intro-inner-circle {
+    @extend .inner-circle;
+    background: #00F977;
+}
+
+.malaria-inner-circle {
+    @extend .inner-circle;
+    background: #EB5A61;
+}
+
+.med-inner-circle {
+    @extend .inner-circle;
+    background: #017E84;
+}
+
+.pc-policy-inner-circle {
+    @extend .inner-circle;
+    background: #F2AE3E;
+}
+
+.malaria-outer-circle {
+    @extend .outer-circle;
+    background: #E13946;  
+    top: $outer-circle-top-pos+20; 
+}
+
+.med-outer-circle {
+    @extend .outer-circle;
+    background: #07717E;
+    top: $outer-circle-top-pos+40;
+}
+
+.pc-policy-outer-circle {
+    @extend .outer-circle;
+    background: #F09938;
+    top: $outer-circle-top-pos+60;
+}
+
+.unlocked-outer-circle {
+    @extend .outer-circle;
+    background: #fff;
+    top: $outer-circle-top-pos+80;
+}
+
+.activity {
+    border-radius: 200px;
+    border: 6px solid transparent;
+    height: 75px;
+    width: 75px;
+    position: absolute;  
+}
+
+.activity1 {
+    @extend .activity;
+    left: 3%;
+    top: 7%;    
+}
+
+.activity1-incomplete {
+    @extend .activity1;
+    border-top-color: #BEBEBE;
+}
+
+.activity1-complete {
+    @extend .activity1;
+    border-top-color: #00F977;    
+}
+
+.activity2 {
+    @extend .activity;
+    left: -2%;
```

Comment:
I tried using display: inline-block and float properties. I also tried changing to relative positioning but absolute positioning was the best possible responsive design since there is overlap of elements.
Also I'm using % values which will scale to the size of the screen.
Can you suggest any other way that will eliminate absolute positioning?